pkg/dns/client/dns_test.go

```diff
@@ -419,7 +444,14 @@ func initDNS(t test.Failer) *LocalDNSServer {
 	}
 	testAgentDNS.resolvConfServers = []string{srv}
 	testAgentDNS.StartDNS()
-	testAgentDNS.searchNamespaces = []string{"ns1.svc.cluster.local", "svc.cluster.local", "cluster.local"}
+	testAgentDNS.searchNamespaces = []string{
```

Comment:
searchNamespaces comes from k8s configuring /etc/resolv.conf. In real world, how is this going to work?

Reply:
I didn't actually need this change.